dephash.py

```diff
@@ -210,8 +210,10 @@ def outdated(virtualenv, path):
 @click.option("--virtualenv", default='virtualenv', help="Path to virtualenv command")
 @click.option("-o", "--output-file", metavar="<requirements_prod>", type=click.Path(),
               help="Specify a file to write to; otherwise output to STDOUT")
+@click.option("-a", "--algorithm", default="sha512",
+              help="Hash algorithm to use, passed to hashin")
```

Comment:
We may want to restrict to `("sha512", "sha256")` via [`click.Choice`](http://click.pocoo.org/5/options/#choice-options)

Reply:
Hashin should break on unknown, so I think this is ok.